### PR TITLE
feat(i18n): add cs-CZ translation

### PR DESCRIPTION
**Pull request by mychalvlcek**

### CODE REVIEW

**Comment by SebastienGllmt** [comment]
On CrowdIn, it looks like Czech translation hasn't been finished so I would avoid merging this PR until then

File: src/i18n/locales/cs-CZ.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "components.common.languagepicker.chinese": "简体中文",
   "components.common.languagepicker.continueButton": "Choose language",
   "components.common.languagepicker.contributors": "_",
+  "components.common.languagepicker.czech": "Čeština",
```
Comment (v-almonacid) [inline src/i18n/locales/cs-CZ.json]: All the `.../locales/xx-XX.json` files, except for `en-US.json`, shouldn't be touched—these are automatically generated by crowdin.

**Review by v-almonacid — CHANGES_REQUESTED** [review]
Thanks for your contribution! Apart from the detail I pointed out, it looks good. We'll definitely merge this once we have enough translations in crowdin.

**Comment by mychalvlcek** [comment]
@SebastienGllmt just finished translation

**Comment by v-almonacid** [comment]
> @SebastienGllmt just finished translation

mhm what do you mean? I just checked CrowdIn and Czech is at 0%. Would you like to join and add the translations yourself? 😄

**Comment by mychalvlcek** [comment]
> > @SebastienGllmt just finished translation
> 
> mhm what do you mean? I just checked CrowdIn and Czech is at 0%. Would you like to join and add the translations yourself? 😄


yes, thats what I did (or thought I did :D ),

is this correct link, right?

https://crowdin.com/project/yoroi-app/cs

**Comment by v-almonacid** [comment]
> > > @SebastienGllmt just finished translation
> > 
> > 
> > mhm what do you mean? I just checked CrowdIn and Czech is at 0%. Would you like to join and add the translations yourself? 😄
> 
> yes, thats what I did (or thought I did :D ),
> 
> is this correct link, right?
> 
> https://crowdin.com/project/yoroi-app/cs

oh that's the Yoroi extension. Yoroi mobile has a separate Crowdin project https://crowdin.com/project/yoroi-mobile

**Comment by mychalvlcek** [comment]
> > > > @SebastienGllmt just finished translation
> > > 
> > > 
> > > mhm what do you mean? I just checked CrowdIn and Czech is at 0%. Would you like to join and add the translations yourself? 😄
> > 
> > 
> > yes, thats what I did (or thought I did :D ),
> > is this correct link, right?
> > https://crowdin.com/project/yoroi-app/cs
> 
> oh that's the Yoroi extension. Yoroi mobile has a separate Crowdin project https://crowdin.com/project/yoroi-mobile

OK, don't have permissions to see this project, could you grant me somehow?

**Comment by v-almonacid** [comment]
@mychalvlcek sure just send me an email address, or join our discord and I'll send you the invite there https://discord.gg/pstXdg5R

**Comment by mychalvlcek** [comment]
Finally translation should be finished (except of terms of use)

**Review by v-almonacid — APPROVED** [review]

**Comment by v-almonacid** [comment]
thank you @mychalvlcek ! note that if you want,  you can put your name in the `components.common.languagepicker.acknowledgement` key in crowdin, and your name/nickname will be in the initial screen.

**Pull request merged**